optaplanner-website-root/data/events.yml

```diff
@@ -16,9 +16,12 @@
 
 - eventId: 20220825-communitycentral
   eventOrganization: "Community Central"
-  eventUrl: https://primetime.bluejeans.com/a2m/live-event/grurbjbd
+  eventUrl: http://communitycentral.tv/
```

Comment:
We ban the use of http url's in favor of https url, because google penalize SEO ranking on the use of HTTP instead of HTTPS